services/web-app/package.json

```diff
@@ -16,7 +16,7 @@
     "lint": "next lint",
     "service:build": "npm run build",
     "start": "next start",
-    "start-secure": "sh ./scripts/create-local-certificates.sh && node server.js",
+    "start-secure": "sh ./scripts/create-local-certificates.sh && RUNNING_SECURELY=1 NODE_TLS_REJECT_UNAUTHORIZED=0 node server.js",
```

Comment:
minor: consider renaming the npm script to `start:secure` to match our other services.